Unix/tests/http/test_http.cpp

```diff
@@ -185,14 +185,16 @@ static void* MI_CALL _http_client_proc(void* param)
     // read response
     char r_buf[1024];
     size_t read = 0;
+    int err;
 
     do
     {
         r = Sock_Read(sock, r_buf, sizeof(r_buf), &read);
+        err = Sock_GetLastError();
     }
-    while (r != MI_RESULT_OK && Sock_GetLastError() == 11 /*EAGAIN*/);
+    while (r != MI_RESULT_OK && (err == 11 || err == 35) /*EAGAIN and EDEADLK*/);
```

Comment:
This isn't quite what I had in mind.

Can't you include errlog.h and then do something like:

`while (r != MI_RESULT_OK && (err == EAGAIN || err == EDEADLK));`

That's more self documenting ...